### DISCUSSION TITLE
Question about healing process

### DISCUSSION THREAD

**Discussion by AlexZIX** [minio/minio#19904] [Q&A] [answered]
Hi.

I've replaced few disks in my minio cluster and now watching the healing process using following command:

mc admin heal minio-cold

And i have few questions about output:

Objects Healed: 2,765,357/1,688,393 (100%%), 12 TiB/14 TiB (83.6%%)
Objects Failed: 2,990/1,688,393 (0.1%%), 41 GiB/14 TiB (0.2%%)
Heal rate: 2 obj/s, 9.0 MiB/s
Estimated Completion: 1 day

1 of 1 sets exceeds reduced parity count EC:2 lost/offline disks

1. Objects Healed: 2,765,357/1,688,393. 
I thought that 2,765,357 is count of healed objects and 1,688,393 is a total objects count. But in this case why 1st greater than 2nd?

2. Objects Failed: 2,990/1,688,393 (0.1%%), 41 GiB/14 TiB (0.2%%). 
Does it means that I have 2990 objects which can't be healed and total size of this objects is 41 GB?

3. 1 of 1 sets exceeds reduced parity count EC:2 lost/offline disks. 
I have 4 nodes with 4 disks on each. EC = 6, reduced EC = 2. I've replaced 2 disks and this string means that I lost few objects with reduced redundancy, right?

**Comment by klauspost** [comment, marked as answer]
1. Numbers are estimates.
2. Maybe, or they could be leftover objects from incomplete deletes or files being deleted after listing, before they could be healde.
3. Could be, yes.

**Comment by AlexZIX** [comment]
Thanks.
